test/javascripts/components/load-more-test.js.es6

```diff
@@ -0,0 +1,28 @@
+import { _setTestParams } from "discourse/lib/eyeline";
+import componentTest from "helpers/component-test";
+
+moduleForComponent("load-more", { integration: true });
+
+componentTest("preview updates with markdown", {
+  template: `
+    {{#load-more selector=".numbers tr" action=(action loadMore)}}
+      <table class='numbers'>
+        <tr><td>1</td></tr>
+        <tr><td>2</td></tr>
+        <tr><td>3</td></tr>
+      </table>
+    {{/load-more}}`,
+
+  beforeEach() {
+    this.set("loadMore", () => this.set("loadedMore", true));
+    _setTestParams(false, true);
```

Comment:
I would prefer a hash, you have to dig to know what are these `false` `true` atm, something like:

```
configureEyeLineBehavior({foo: true, bar: false});
```

Would be better I think. I also changed the name, because it's used all the time and not only in tests now, even if you created it for tests, its real goal is to tweak eye line behaviour.

Reply:
Also I'm wondering if we couldn't use: `Discourse.rootElement` to have all of this automatically working?

Reply:
This makes sense. I implemented the `configureEyeline` function.

Regarding the other comment, `.position()` and `.offset()` return slightly different results (relative to parent vs document) and I recommend this to be used only during testing.